feature-extensions/apigee/setup.py

```diff
@@ -63,12 +66,14 @@ def main():
         help='Specify the scm repo path ext (Use "scm" for bitbucket)'
     )
 
-    mainParser.add_argument("apigee-host", help="Url of the Apigee host (e.g. https://my-apigee-host)")
-    mainParser.add_argument("apigee-org", help="Name of the Apigee org you wish to use")
-    mainParser.add_argument("apigee-env", help="Name of the Apigee env you wish to use")
-    mainParser.add_argument("apigee-build", help="Version to stamp Apigee proxy with (TODO why does the user need to provide this?)")
-    mainParser.add_argument("apigee-username", help="Username to use when accessing Apigee")
-    mainParser.add_argument("apigee-password", help="Password to use when accessing Apigee")
+    mainParser.add_argument("apigee_host", help="Url of the Apigee host (e.g. https://my-apigee-host)")
+    mainParser.add_argument("apigee_org", help="Name of the Apigee org you wish to use")
+    mainParser.add_argument("apigee_env", help="Name of the Apigee env you wish to use")
+    mainParser.add_argument("apigee_svc_host", help="Url of the service API host (e.g. jazz.api.t-mobile.com)")
+    mainParser.add_argument("apigee_build", help="Version to stamp Apigee proxy with \
+                            (TODO why does the user need to provide this?)")
```

Comment:
Did we ever answer this question?

Reply:
I think, we dont need this as user input.

Reply:
@rajeevr2715 I agree, let's remove it.

Reply:
Done